main/test/ca/uwaterloo/flix/TestVerifiers.scala

```diff
@@ -0,0 +1,28 @@
+package ca.uwaterloo.flix
+
+import ca.uwaterloo.flix.api.{Flix, FlixEvent, FlixListener}
+import ca.uwaterloo.flix.util.Options
+import ca.uwaterloo.flix.verifier.{TokenVerifier, TypeVerifier}
+import org.scalatest.funsuite.AnyFunSuite
+
+class TestVerifiers extends AnyFunSuite with TestUtils {
+
+  test("RunAllVerifiers") {
```

Comment:
I think its important to have one test that just runs all the verifiers (or one for each verifier) to make sure that they are always run at least once